spyvm/primitives.py

```diff
@@ -608,8 +608,9 @@ def func(interp, s_frame, w_frame, stackp):
     w_frame.store(interp.space, constants.CTXPART_STACKP_INDEX, interp.space.wrap_int(stackp))
     return w_frame
 
-def get_instances_array(space, s_frame, w_class):
+def get_instances_array(space, s_frame, w_class=None):
     # This primitive returns some instance of the class on the stack.
+    # If no class is given, it returns some object.
     # Not sure quite how to do this; maintain a weak list of all
     # existing instances or something?
     match_w = s_frame.instances_array(w_class)
```

Comment:
Yes. In case `w_class is None`, we get all instances of all objects (except for `SmallInteger`).